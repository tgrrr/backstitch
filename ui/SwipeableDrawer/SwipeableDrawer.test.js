import React from 'react'
import renderer from 'react-test-renderer'
import { SwipeableDrawer } from '.'

it('SwipeableDrawer: default', () => {
    const component = renderer.create(
        <SwipeableDrawer
            onOpen={() => { }}
            onClose={() => { }}
            open={true}
        >
            <>
                SwipeableDrawer
            </>
        </SwipeableDrawer>
    )
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
