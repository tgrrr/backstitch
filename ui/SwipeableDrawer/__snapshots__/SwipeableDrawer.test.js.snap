// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SwipeableDrawer: default 1`] = `
<div
  className="sc-AxjAm StDqN"
>
  <div
    className="Swipeabledrawer "
    data-testid="Swipeabledrawer"
  />
</div>
`;
